backend/inspirehep/indexer/cli.py

```diff
@@ -334,3 +334,30 @@ def create_aliases(ctx, yes_i_know, prefix_alias):
             current_search.client.indices.delete_alias("*", alias_name)
         click.echo(f"Creating alias '{alias_name}' -> '{index_name}'")
         current_search.client.indices.put_alias(index=index_name, name=alias_name)
+
+
+@index.command(
+    "delete-indexes", help="Removes indexes with specified prefix",
+)
+@click.option("--yes-i-know", is_flag=True)
+@click.option("--prefix", default="", type=str)
+@with_appcontext
+@click.pass_context
+def delete_aliases(ctx, yes_i_know, prefix):
+    indices = current_search.client.indices.get_alias()
+    prefix_regex = re.compile(f"""{prefix}.*""")
+    indices_to_delete = list(filter(prefix_regex.match, indices))
+    indices_to_delete = dict((k, indices[k]) for k in indices_to_delete)
```

Comment:
`indices_to_delete = {k:indices[k] for k in indices_to_delete}` 
so you won't have to build it twice (first as tuple and then dict from this tuple)